Digipost-Test-Dpost Tests/OAuthTests.swift

```diff
@@ -44,34 +45,41 @@ class OAuthTests: XCTestCase {
         let token = OAuthToken(attributes: oAuthDictionary, scope: scope)
         return token!
     }
-    
+
     func testOauthFromDictionary() {
```

Comment:
agreed, i'll add more specific names. For example testCreateOAuthTokenFromJsonDictionary